src/components/metrics/card/name/index.js

```diff
@@ -1,108 +1,117 @@
 import React, {Component, PropTypes} from 'react'
 
-import TextArea from 'react-textarea-autosize'
+import {EditorState, Editor, ContentState, getDefaultKeyBinding, KeyBindingUtil} from 'draft-js'
 
 import './style.css'
 
+class SimpleEditor extends React.Component {
+  state = {
+    editorState: this._plainTextEditorState(this.props.value)
+  }
+
+  _plainTextEditorState(value) {
+    return EditorState.createWithContent(ContentState.createFromText(value || ''))
+  }
+
+  _onChange(editorState) {
+   return this.setState({editorState})
+  }
+
+  focus() {
+    this.refs.editor.focus()
+  }
+
+  changePlainText(value) {
+    this.setState({editorState: this._plainTextEditorState(value)})
```

Comment:
you did something pretty similar a different way in the previous SimpleEditor component. Why do them different ways?